packages/compass-connections/src/components/resizeable-sidebar.tsx

```diff
@@ -1,84 +1,57 @@
-import React, { useState } from 'react';
+import React, { useCallback, useState } from 'react';
 import {
   ResizeHandle,
   ResizeDirection,
   uiColors,
   css,
-  spacing,
 } from '@mongodb-js/compass-components';
-import type { ConnectionInfo } from 'mongodb-data-service';
-
-import ConnectionList from './connection-list/connection-list';
-
-const initialSidebarWidth = spacing[4] * 10 + spacing[2]; // 248px
-const minSidebarWidth = spacing[4] * 7; // 168px
 
 const listContainerStyles = css({
   display: 'flex',
   flexDirection: 'column',
   margin: 0,
   padding: 0,
   maxWidth: '80%',
-  minWidth: minSidebarWidth,
   height: '100%',
   position: 'relative',
   background: uiColors.gray.dark3,
   color: 'white',
 });
 
-function getMaxSidebarWidth() {
-  return Math.max(minSidebarWidth, window.innerWidth - 100);
-}
-
-function ResizableSidebar({
-  activeConnectionId,
-  connections,
-  createNewConnection,
-  setActiveConnectionId,
-  onConnectionDoubleClicked,
-  removeAllRecentsConnections,
-  duplicateConnection,
-  removeConnection,
+const ResizableSidebar = ({
+  initialWidth,
+  minWidth,
+  children,
 }: {
-  activeConnectionId?: string;
-  connections: ConnectionInfo[];
-  createNewConnection: () => void;
-  setActiveConnectionId: (newConnectionId: string) => void;
-  onConnectionDoubleClicked: (connectionInfo: ConnectionInfo) => void;
-  removeAllRecentsConnections: () => void;
-  duplicateConnection: (connectionInfo: ConnectionInfo) => void;
-  removeConnection: (connectionInfo: ConnectionInfo) => void;
-}): React.ReactElement {
-  const [width, setWidth] = useState(initialSidebarWidth);
+  initialWidth: number;
+  minWidth: number;
+  children: JSX.Element;
+}): JSX.Element => {
+  const [width, setWidth] = useState(initialWidth);
+
+  const getMaxSidebarWidth = useCallback(() => {
+    return Math.max(minWidth, window.innerWidth - 100);
+  }, [minWidth]);
 
   return (
     <div
       className={listContainerStyles}
       style={{
-        minWidth: width,
+        minWidth: minWidth,
```

Comment:
I'm not sure if it's this change, but it looks like the sidebar now defaults to a narrower width:
(beta below, this branch top) - is that intended?
<img width="340" alt="Screen Shot 2022-03-16 at 12 59 57 PM" src="https://user-images.githubusercontent.com/1791149/158647297-97beb081-a97a-4d97-a29d-0ecac896ae0e.png">

Reply:
Yeah, you're right! Will fix it